src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
As far as docs, the page for `.on()` already says that for native events there's an `originalEvent` property. The only docs clarification there I can think of is to add an `Event.originalEvent` page and make it explicitly clear that the property may or may not be there. The concept of an "event chain" isn't something that jQuery Core depends upon, it's just an accident of the way `jQuery.Event` works.

If you look at the problem from the standpoint of how well `.simulate()`d a *native* event is, then the best approach seems to use `new jQuery.Event(type)` because no natively generated event would be missing an `originalEvent`. However, the only place we're simulating events in master is for focusin/focusout, we've never gotten a report of problems the way it's being done now, and this whole simulation mess is hopefully not going to be used a lot in the future. These are all really rare edge cases so I doubt it really matters what we do here--or if it does matter there is no way we'll find out except to change the code.